rich/text.py

```diff
@@ -1204,7 +1204,7 @@ def wrap(
 
         Args:
             console (Console): Console instance.
-            width (int): Number of characters per line.
+            width (int): Number of single-width characters per line.
```

Comment:
Technically correct but a bit awkward, no?
How about the suggestion below or something of the sorts?

```suggestion
            width (int): Number of cells available per line.
```